pkg/runtime/optimize/dal/show_open_tables.go

```diff
@@ -55,15 +55,34 @@ func optimizeShowOpenTables(ctx context.Context, o *optimize.Optimizer) (proto.P
 
 	tenant := rcontext.Tenant(ctx)
 	clusters := security.DefaultTenantManager().GetClusters(tenant)
+
+	invertedDatabases := make(map[string]string)
```

Comment:
cap add clusters.len

Reply:
this cap cannot be certain at this time, it will be the sum of all dbgroups